src/config/config.ts

```diff
@@ -21,19 +21,21 @@ const fornoApiKey = process?.env?.NEXT_PUBLIC_FORNO_API_KEY || '';
 const celoscanApiKey = process?.env?.NEXT_PUBLIC_CELOSCAN_API_KEY || '';
 const infuraApiKey = process?.env?.NEXT_PUBLIC_INFURA_API_KEY || '';
 const upstashKey = process?.env?.UPSTASH_KEY || '';
+const useAlfajores = process?.env?.NEXT_PUBLIC_USE_ALFAJORES === '1' || false;
```

Comment:
because hooks begin with use. it might be better to use a different name. evening usingAlfajores or isAlfajores